al_aws_collector.js

```diff
@@ -151,7 +151,7 @@ class AlAwsCollector {
                         util.inspect(error);
             }
             const status = this.prepareErrorStatus(errorString);
-            this.sendStatus(status, () => {
+            this.sendStatus(status, (err) => {
```

Comment:
Should we handle that sending error?

Though handling the error handlers' error is making my head hurt.

Reply:
Removed `err`